lib/ramen_bar.rb

```diff
@@ -0,0 +1,34 @@
+class RamenBar
+
+  attr_accessor :dishes, :order, :total_order
+
+  MENU = {
+    'tonkotsu' => 10,
+    'kakugiri' => 9,
+    'shoyu'    => 9,
+    'gyoza'    => 5,
+    'edamame'  => 3
+  }
+
+  def initialize(dishes = MENU)
+    @dishes = dishes
+    @order  = []
+    @total_order = 0
+  end
+
+  def menu
+    @dishes.map { |dish, price| "#{dish} - £#{price}" }.join(', ')
```

Comment:
Nice chaining of the methods to produce a menu! Suggest maybe joining them with '\n' instead to have every item on a different line.